tests/smtp-long-DATA-line-03-frames-ips/test.yaml

```diff
@@ -0,0 +1,28 @@
+requires:
+  min-version: 8
+
+args:
+- -k none
+- --simulate-ips
+
+checks:
+- filter:
+    count: 1
+    match:
+      alert.signature_id: 1
+- filter:
+    count: 2 # TODO 2 because of multiple smtp.data updates, and we're doing sliding window
```

Comment:
Should one frame match only once per signature as is already done for transactions ?

Reply:
maybe, but that is out of scope for this work. It's a general streaming buffer issue.

Reply:
Ok

Or if a frame matches and there is a tx associated to it, we can add the sid in the engine state there...